src/components/refactor/dialogs/generator-creation/reactive-capability-curve/reactive-capability-curve-row-form.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import FloatInput from '../../../rhf-inputs/float-input';
+import {
+    P,
+    Q_MAX_P,
+    Q_MIN_P,
+    REACTIVE_CAPABILITY_CURVE_TABLE,
+} from '../../../utils/field-constants';
+import {
+    ActivePowerAdornment,
+    gridItem,
+    ReactivePowerAdornment,
+} from '../../../../dialogs/dialogUtils';
+import React from 'react';
+
+const ReactiveCapabilityCurveRowForm = ({ index, labelSuffix }) => {
+    const id = REACTIVE_CAPABILITY_CURVE_TABLE;
+    const pField = (
+        <FloatInput
+            name={`${id}.${index}.${P}`}
+            label={'P'}
+            labelValues={{ labelSuffix: labelSuffix }}
```

Comment:
```suggestion
            label={'P' + labelSuffix}
```
This is the way labels are done in main branch, it will even remove the need of labelValues prop
Except if there is a use case I don't understand here

Reply:
When there is over two rows, we don't translate the suffix which cause warnings.
This way the suffix is taken dynamically every time without warning.